firmware/screen/src/ui/screen_pocket_technician.cpp

```
@@ -0,0 +1,450 @@
+
+    return scr;
+}
+
+extern "C" void screen_pocket_technician_clear_setups(void) {
+    g_setup_count = 0;
+    if (g_active_ctx) {
+        rebuild_list_widgets(g_active_ctx);
+    }
+}
+
+extern "C" void screen_pocket_technician_add_setup(const char* name,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'screen_pocket_technician_add_setup' of similar type ('const char *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)